docs/rules/no-empty-fields.md

```diff
@@ -33,3 +33,22 @@ Example of **correct** code for this rule:
 	}
 }
 ```
+
+## Options
+
+### `ignoreProperties`
+
+Pass an array of top-level package properties to ignore.
+When provided, any errors related to the properties won't cause the rule to report a violation.
```

Comment:
[Docs] Nit: "errors" are a by-product of rule report violations, not the other way around. ✨ 

```suggestion
When provided, any of those properties being empty won't cause the rule to report a violation.
```